server/requirements.txt

```diff
@@ -1,6 +1,9 @@
+alembic>=1.6.4
+aniso8601>=9.0.1
 boto3
+click
 python-dateutil
-elasticsearch
+elasticsearch==7.13.1
```

Comment:
While this is expedient and _probably_ what we want, pinning us to a specific version seems overly restrictive, which might bite us later.

I recommend `=>7.13.0` and `<7.14.0`, which I think better expresses our requirements.

Reply:
The idea was simply that we knew 7.13.1 worked and didn't want to iterate through any intervening versions to determine where the package problem was introduced.

I think that in terms of general configuration management we'd be better off pinning everything to a specific version we knew meets our requirements. The downside is that we need periodic tasks to evaluate newer versions to determine when to upgrade. In general we've avoided that by letting them float except where we encounter problems. Not the best strategy in the long run, but expedient.

Reply:
Perhaps its needless paranoia, but I'm imagining `7.13.1` being superseded by `7.13.2` and becoming unavailable, leaving our installation broken.  So, my solution to that is to lock out the version that we know doesn't work, while still affording some flexibility.

(Hmmm, I wonder if `!=7.14.0` is an available and workable option....)